### ISSUE TITLE
Plugin cannot be activated due to critical error

### ISSUE THREAD

**Issue by dotherightthing** [dotherightthing/wpdtrt-form#10]
WordPress version 5.6
Current theme: WPDTRT DBTH (version 3.7.1)
Current plugin: DTRT Form (version 0.3.0)
PHP version 7.2.22

```
Error Details
=============
An error of type E_ERROR was caused in line 1152 of the file /container/application/public/wp-content/plugins/wpdtrt-form/vendor/dotherightthing/wpdtrt-plugin-boilerplate/src/Plugin.php. Error message: Uncaught Error: Cannot use object of type WP_Error as array in /container/application/public/wp-content/plugins/wpdtrt-form/vendor/dotherightthing/wpdtrt-plugin-boilerplate/src/Plugin.php:1152
Stack trace:
#0 /container/application/public/wp-content/plugins/wpdtrt-form/src/class-wpdtrt-form-plugin.php(378): DoTheRightThing\WPDTRT_Plugin_Boilerplate\r_1_7_15\Plugin->get_api_data()
#1 /container/application/public/wp-content/plugins/wpdtrt-form/src/class-wpdtrt-form-plugin.php(354): WPDTRT_Form_Plugin->helper_sendmail('contact-form', 'Contact', Array)
#2 /container/application/public/wp-includes/class-wp-hook.php(287): WPDTRT_Form_Plugin->helper_sendmail_proxy('')
#3 /container/application/public/wp-includes/class-wp-hook.php(311): WP_Hook->apply_filters(NULL, Array)
#4 /container/application/public/wp-includes/plugin.php(484): WP_Hook->do_action(Array)
#5 /container/application/public/wp-settings.php(557): do_action('init')
#6 /container/application/public/wp-config.php(118): require_once('/container/appl...')
```

**Comment by dotherightthing** [comment]
See dotherightthing/wpdtrt-plugin-boilerplate#189 and dotherightthing/wpdtrt-npm-scripts#50

**Comment by dotherightthing** [comment]
Resolved with dotherightthing/wpdtrt-npm-scripts#50

dotherightthing/wpdtrt-plugin-boilerplate#189 still needs to be actioned at some point.
